### ISSUE TITLE
No provider for Ng2FrameworkFactory

### ISSUE THREAD

**Issue by jackyphuong** [ceolter/ag-grid-ng2#42]
Hello,

I just update to latest version 6.0.2 and meet this issue No provider for Ng2FrameworkFactory

core.umd.js:3462 EXCEPTION: Uncaught (in promise): Error: Error in client/administration/auditlog.component.html:10:20 caused by: No provider for Ng2FrameworkFactory!

It is working well on previous version

Please help me check it

Thanks,

**Comment by mcwillso** [comment]
This is resolved by including the AgGridModule.forRoot() in your app module:
```
import {AgGridModule} from "ag-grid-ng2/main";
```
then in your imports:
```
imports: [
    ...
    AgGridModule.forRoot()
  ],
```

**Comment by jackyphuong** [comment]
Confirmed, it works well now. Thanks

**Comment by brannon-darby** [comment]
resolved in latest release (currently 7.2.2) by including AgGridModule.withComponents([...components...])

```
import { AgGridModule } from "ag-grid-ng2/main";

...

imports: [
  ...
  AgGridModule.withComponents([
    ...components...
  ])
]
```
